Relevant traceback from prior to this change:


```
tests/conftest.py:77: in plugin
    await instance.internal_start()
.tox/py311/lib64/python3.11/site-packages/maubot/plugin_base.py:110: in internal_start
    self.register_handler_class(self)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <fedora.Fedora object at 0x7f30439fa050>, obj = <fedora.Fedora object at 0x7f30439fa050>

    def register_handler_class(self, obj) -> None:
        warned_webapp = False
        for key in dir(obj):
>           val = getattr(obj, key)
                  ^^^^^^^^^^^^^^^^^
E           AttributeError: __provides__

.tox/py311/lib64/python3.11/site-packages/maubot/plugin_base.py:81: AttributeError

```

--- a/fedora/fedmsg.py
+++ b/fedora/fedmsg.py
@@ -29,7 +29,7 @@ def _publish(message: message.Message):
     api.publish(message)
 
 
-_background_tasks = set()
+_background_tasks: set[asyncio.Future] = set()
 
 
 async def publish(message: message.Message):

